mantis-common/src/main/java/io/mantisrx/common/metrics/measurement/Measurements.java

```diff
@@ -46,6 +48,21 @@ public Measurements(
         this.tags = tags;
     }
 
+    @JsonCreator
```

Comment:
instead of adding a new method, please add a new `Collection<MicrometerMeasurements>` arg to the original constructor and refactor accordingly.